docs/resources/application_agent_credential.md

```diff
@@ -12,7 +12,7 @@ description: |-
 
 ## Example Usage
 
-{{tffile "/home/runner/work/terraform-provider-indykite/terraform-provider-indykite/examples/resources/indykite_application_agent_credential/resource.tf"}}
+{{tffile "/Users/zhekaby/src/terraform-provider-indykite/examples/resources/indykite_application_agent_credential/resource.tf"}}
```

Comment:
This seems weird. Something is probably wrong with generator.

But it seems it didn't work before either: https://registry.terraform.io/providers/indykite/indykite/latest/docs/resources/application_agent_credential

We should take a look

Reply:
should I revert that chunk? generator created that

Reply:
No, it will not work with your change, nor previous state. We need to find a proper fix